dotnet-roslyn/analyzers/Audacia.CodeAnalysis.Analyzers/src/Audacia.CodeAnalysis.Analyzers/Common/HelpLinkUrlFactory.cs

```diff
@@ -52,6 +52,10 @@ public static string Create(string diagnosticId)
                     return $"{helpLinkUrlPrefix}use-record-types";
                 case DiagnosticId.DoNotUseNumberInIdentifierName:
                     return $"{helpLinkUrlPrefix}do-not-include-numbers-in-identifier-name";
+                case DiagnosticId.DoNotUseProducesResponseTypeWithTypedResults:
+                    return $"{helpLinkUrlPrefix}do-not-use-producesresponse-attribute-with-typedresults";
```

Comment:
this also doesn't seem to match the title for ACL1015

Reply:
fixed this too